src/metabase/api/dashboard.clj

```diff
@@ -494,6 +509,50 @@
       (u/prog1 (first (last-edit/with-last-edit-info [dashboard] :dashboard))
         (events/publish-event! :event/dashboard-read {:object-id (:id dashboard) :user-id api/*current-user-id*})))))
 
+(api/defendpoint GET "/:id/items"
+  "Get Dashboard with ID."
+  [id]
+  {id ms/PositiveInt}
+  ;; query copied from metabase.api.collection to match the shape of api/collection/<:id|root>/items
```

Comment:
Do they need to kinda be kept in sync? If so I'd add it to the comment

Reply:
Makes sense, will do. There is also a test that asserts that the shape is the same, so if someone misses the comment we'll let them know.